src/astro/sql/operators/sql_decorator.py

```diff
@@ -218,29 +218,22 @@ def handle_output_table_schema(self, output_table_name, schema=None):
         :return:
         """
         schema = schema or get_schema()
-        if self.conn_type == "postgres" and self.schema:
+        if self.conn_type in ["postgres", "bigquery"] and self.schema:
```

Comment:
I get the impression we are doing this type of manipulation in several parts of the code. Could we replace this function by using `Table.qualified_name` (https://github.com/astro-projects/astro/blob/ab9efd134799129985622846236a50bbd8e6add2/src/astro/sql/table.py#L57)